### PR TITLE
add dicom reader function and pdf visualization

### PR DESCRIPTION
**Pull request by johnccshen**

### CODE REVIEW

**Comment by johnccshen** [comment]
Let's merge Stefan's code first and then I'll rebase to master to update the code

**Comment by johnccshen** [comment]
@cepa995 I made some changes to the dicom reader, could you take a look

File: toolbox/dicom.py [hunk 1]
```diff
@@ -170,10 +170,35 @@ def load_scan(dicom_dir_path):
...
-    return itk_image
\ No newline at end of file
+        :param dcm_root: a folder path, it should be the parent folder of a dicom series,
+        for example /kaggle/input/rsna-miccai-brain-tumor-radiogenomic-classification/train/00000/FLAIR/.
+        """
+        self.dcm_root = dcm_root
+        self.itk_image = self.read_from_dicom_series()
+
+    def get_array(self):
+        return sitk.GetArrayFromImage(self.itk_image)
```
Comment (lwjason) [inline toolbox/dicom.py:185]: why doing this ha ha. It's not that valuable, to be honest, but we are not building sdintegration, so totally fine!

Comment (lwjason) [reply toolbox/dicom.py:185]: I saw how it's used below. It looks fine!

**Review by lwjason — APPROVED** [review]

**Pull request merged**